#!/usr/bin/env python2
PACKAGE = "snd_control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("left_speed_p", double_t, 1, "left Motor Speed P", 1, 0, 4)
gen.add("left_speed_i", double_t, 1, "left Motor Speed I", 0, 0, 4)
gen.add("left_speed_d", double_t, 1, "left Motor Speed D", 0, 0, 1)
gen.add("right_speed_p", double_t, 2, "right Motor Speed P", 1, 0, 4)
gen.add("right_speed_i", double_t, 2, "right Motor Speed I", 0, 0, 4)
gen.add("right_speed_d", double_t, 2, "right Motor Speed D", 0, 0, 1)

exit(gen.generate(PACKAGE, "snd_control", "Pid"))
